pkg/process/metadata/workloadmeta/grpc_test.go

```diff
@@ -308,3 +333,30 @@ func toEventSet(proc *procutil.Process) *pbgo.ProcessEventSet {
 func toEventUnset(proc *procutil.Process) *pbgo.ProcessEventUnset {
 	return &pbgo.ProcessEventUnset{Pid: proc.Pid}
 }
+
+// setupGRPCTest a test extractor, server, and client connection.
+// Cleanup is handled automatically via T.Cleanup().
+func setupGRPCTest(t *testing.T) (*WorkloadMetaExtractor, *GRPCServer, *grpc.ClientConn, pbgo.ProcessEntityStream_StreamEntitiesClient) {
+	cfg := config.Mock(t)
+	port, err := testutil.FindTCPPort()
+	require.NoError(t, err)
+	cfg.Set("process_config.language_detection.grpc_port", port)
+	extractor := NewWorkloadMetaExtractor(cfg)
+
+	grpcServer := NewGRPCServer(cfg, extractor)
+	err = grpcServer.Start()
+	require.NoError(t, err)
+	t.Cleanup(grpcServer.Stop)
+
+	cc, err := grpc.Dial(grpcServer.addr.String(), grpc.WithTransportCredentials(insecure.NewCredentials()))
+	require.NoError(t, err)
+	t.Cleanup(func() {
+		err := cc.Close()
+		require.NoError(t, err)
```

Comment:
Do we really need to check of error here? I don't think validating the cleanup matters to the behavior you're trying to test. I think it'll be better to ignore this.

Reply:
Probably not, actually! I actually ran into some trouble before, where if we'd like to close the connection to force an error the test would fail because of this. I didn't actually commit that test, but it was a blocker at one point for me.

I'll go ahead and remove the assertion.